src/include/distributed/multi_client_executor.h

```diff
@@ -72,6 +75,28 @@ typedef enum
 } BatchQueryStatus;
 
 
+/* Enumeration to track what the task associated with a connection is waiting for */
+typedef enum WaitStatus
```

Comment:
On 2016-05-27 14:25:41 -0700, Jason Petersen wrote:
> > @@ -72,6 +75,28 @@ typedef enum
> >  } BatchQueryStatus;
> >  
> >  
> > +/* Enumeration to track what the task associated with a connection is waiting for */
> > +typedef enum WaitStatus
> 
> Looks good. I hesitated to put _just_ the word read or write in the last two values, since I thought the word _need_ or_wait_ would provide more information. But this is better than `WaitStatus`! I think it's better enough.

I think the SOCKET_ prefix makes that mostly clear.